lib/mayaUsd/fileio/shading/shadingModeExporterContext.cpp

```diff
@@ -453,63 +456,77 @@ bool isSurfaceNodeType(const std::string& nodeType)
 
     return std::binary_search(sKnownSurfaces.cbegin(), sKnownSurfaces.cend(), nodeType);
 }
+
+std::string getMaterialName(
```

Comment:
Factored out the code that generates the material name, to make the function below simpler and clearer.

Reply:
Should we rename this function to follow the ones above? _GetMaterialName? Or is there a different logic being applied here?

Reply:
We usually use `_` for private / protected functions.

Reply:
Our coding standard says to name functions with lowercase staring letters. Not all functions are following the standard, we inherited code.